src/services/ProductService.ts

```diff
@@ -2,14 +2,41 @@ import api from '@/api';
 
 const fetchProducts = async (query: any): Promise <any>  => {
```

Comment:
I think we should rename it to `fetchShipments`, as we are getting shipments in the response. Also, in the whole app, we should replace products or orders with shipments, WDYT?

Reply:
Done